src/WCSimConstructCylinder.cc

```diff
@@ -936,8 +936,8 @@ If used here, uncomment the SetVisAttributes(WClogic) line, and comment out the
         G4Box *solidWCBarrelCellODWLSPlate =
 				new G4Box("WCBarrelCellODWLS",
                           WCODWLSPlatesThickness/2,
-						  WCPMTODRadius*3,
-						  WCPMTODRadius*3);
+						  WCODWLSPlatesLength*3,
+						  WCODWLSPlatesLength*3);
 
```

Comment:
What about we remove the 'times3' bit ? Like that the user have full control on the size of the WLS plates.
I'd suggest to replace this WCODWLSPlatesLength*3 by WCODWLSPlatesLength/2, so like that we sure we create a square of WCODWLSPlatesLength size.

Reply:
Yes that is a good idea.